docs/map.xsd

```diff
@@ -59,13 +129,33 @@
   </xs:restriction>
 </xs:simpleType>
 
+  <xs:simpleType name="cornerterrainT">
+    <xs:restriction base="xs:string">
+      <xs:pattern value="[0-9],[0-9],[0-9],[0-9]"/>
```

Comment:
[0-9]{1,3},[0-9]{1,3},[0-9]{1,3},[0-9]{1,3} this would. But it would also allow up to 999,999,999,999 - we shouldn't do value validation in regex, only string restrictions.